scripts/globals/effects/perfect_counter.lua

```diff
@@ -1,14 +1,15 @@
 -----------------------------------
 --
---
+--     tpz.effect.PERFECT_COUNTER
 --
 -----------------------------------
-
 function onEffectGain(target, effect)
+    target:addMod(tpz.mod.PERFECT_COUNTER_ATT, 100)
 end
 
 function onEffectTick(target, effect)
 end
 
 function onEffectLose(target, effect)
+    target:addMod(tpz.mod.PERFECT_COUNTER_ATT, -100)
```

Comment:
delmod with 100, not addmod -100

admod and delmod belong paired

Reply:
changed to delmod 100 not addmod -100